relays/ethereum/src/cli.yml

```diff
@@ -112,13 +126,16 @@ subcommands:
                 value_name: SUB_AUTHORITIES_SET_ID
                 help: ID of initial GRANDPA authorities set.
                 takes_value: true
+                required: true
             - sub-authorities-set:
                 long: sub-authorities-set
                 value_name: SUB_AUTHORITIES_SET
                 help: Encoded initial GRANDPA authorities set.
                 takes_value: true
+                required: true
             - sub-initial-header:
                 long: sub-initial-header
                 value_name: SUB_INITIAL_HEADER
                 help: Encoded initial Substrate header.
                 takes_value: true
+                required: true
```

Comment:
I don't think it should be required - deploy is able to read it from the substrate node.